indico/modules/events/contributions/models/contributions.py

```diff
@@ -331,6 +355,9 @@ def get_non_inheriting_objects(self):
     def get_field_value(self, field_id):
         return next((v.friendly_data for v in self.field_values if v.contribution_field_id == field_id), '')
 
+    def get_field(self, field_id):
```

Comment:
belongs in the mixin, and i think the name is not really good since you get a field value, not a field